docs/doxygen/plantuml/pkcs11_digest.pu

```diff
@@ -0,0 +1,38 @@
+@startuml
```

Comment:
Can you please add `skinparam dpi 300` to all of the images so they are not blurry when rendered?
```suggestion
@startuml
skinparam dpi 300
```

Reply:
I have added this and re-rendered the images